examples/models/yolo12/README.md

```diff
@@ -0,0 +1,118 @@
+# YOLO12 Detection C++ Inference with ExecuTorch
+
+<p align="center">
+      <br>
+      <img src="./yolo12s_demo.gif">
+      <br>
+</p>
+
+This example demonstrates how to perform inference of [Ultralytics YOLO12 family](https://docs.ultralytics.com/models/yolo12/) detection models in C++ leveraging the Executorch backends:
+- [OpenVINO](../../../backends/openvino/README.md)
+- [XNNPACK](../../../backends/xnnpack/README.md)
+
+# Performance Evaluation
+
+| CPU                            | Model   | Backend  | Device | Precision | Average Latency, ms |
+|--------------------------------|---------|----------|--------|-----------|-----------------|
+| Intel(R) Core(TM) Ultra 7 155H | yolo12s | openvino | CPU    | FP32      | 88.3549         |
+| Intel(R) Core(TM) Ultra 7 155H | yolo12s | openvino | CPU    | INT8      | 53.066          |
+| Intel(R) Core(TM) Ultra 7 155H | yolo12l | openvino | CPU    | FP32      | 317.953         |
+| Intel(R) Core(TM) Ultra 7 155H | yolo12s | openvino | CPU    | INT8      | 150.846         |
+| Intel(R) Core(TM) Ultra 7 155H | yolo12s | xnnpack  | CPU    | FP32      | 169.36          |
+| Intel(R) Core(TM) Ultra 7 155H | yolo12l | xnnpack  | CPU    | FP32      | 436.876         |
```

Comment:
Thanks for posting the numbers.

There are no numbers for GPU or accelerator? Does it not work on those?

Also if possible include xnnpack int8 numbers as well.

Reply:
I've added the GPU numbers. The int8 XNNPACK model could not be lowered to XNNPACK, unfortunately. I've created a ticket to cover that https://github.com/pytorch/executorch/issues/11523